src/ceph-volume/ceph_volume/tests/util/test_device.py

```diff
@@ -127,6 +127,36 @@ def test_is_member_blkid(self, device_info, label):
 
         assert disk.is_member is True
 
+    def test_no_removable_device(self, device_info):
```

Comment:
these tests are really good, could you add the opposite condition? For this test it would be that a normal non-removable device would be valid

Reply:
Added a 2nd test to ensure the opposite